Author: Sergei Golovan <sgolovan@debian.org>
Description: Fixes a bug with .RE macro in the Erlang manpages
 and a few bugs with docs installation.
Last-modified: Thu, 09 Dec 2010 10:45:56 +0300

--- erlang-16.b-dfsg.orig/lib/erl_docgen/priv/xsl/db_man.xsl
+++ erlang-16.b-dfsg/lib/erl_docgen/priv/xsl/db_man.xsl
@@ -508,7 +508,7 @@
         <xsl:text>&#10;.RS 2</xsl:text>
         <xsl:text>&#10;.LP&#10;</xsl:text>
         <xsl:value-of select="$content"/>
-        <xsl:text>&#10;.RE</xsl:text>
+        <xsl:text>&#10;.RE&#10;</xsl:text>
       </xsl:otherwise>
     </xsl:choose>
   </xsl:template>
@@ -816,7 +816,7 @@
   <xsl:template match="desc">
     <xsl:text>&#10;.RS</xsl:text>
     <xsl:apply-templates/>
-    <xsl:text>&#10;.RE</xsl:text>
+    <xsl:text>&#10;.RE&#10;</xsl:text>
   </xsl:template>
 
 
--- erlang-16.b-dfsg.orig/lib/megaco/doc/src/Makefile
+++ erlang-16.b-dfsg/lib/megaco/doc/src/Makefile
@@ -172,8 +172,6 @@
 	$(INSTALL_DATA) $(INFO_FILE) "$(RELSYSDIR)"
 	$(INSTALL_DIR) "$(RELEASE_PATH)/man/man3"
 	$(INSTALL_DATA) $(MAN3DIR)/* "$(RELEASE_PATH)/man/man3"
-	$(INSTALL_DIR)  "$(RELSYSDIR)/doc/standard"
-	$(INSTALL_DATA) $(STANDARDS) "$(RELSYSDIR)/doc/standard"
 
 release_spec:
 
--- erlang-16.b-dfsg.orig/lib/stdlib/doc/src/re.xml
+++ erlang-16.b-dfsg/lib/stdlib/doc/src/re.xml
@@ -310,7 +310,7 @@
       metacharacter should not match before it. Setting this without
       <c>multiline</c> (at compile time) causes circumflex never to
       match. This option only affects the behavior of the circumflex
-      metacharacter. It does not affect \A.</item>
+      metacharacter. It does not affect \\A.</item>
 
       <tag><c>noteol</c></tag>
 
@@ -319,8 +319,8 @@
       match it nor (except in multiline mode) a newline immediately
       before it. Setting this without <c>multiline</c> (at compile time)
       causes dollar never to match. This option affects only the
-      behavior of the dollar metacharacter. It does not affect \Z or
-      \z.</item>
+      behavior of the dollar metacharacter. It does not affect \\Z or
+      \\z.</item>
 
       <tag><c>{offset, integer() >= 0}</c></tag>
 
--- erlang-16.b-dfsg.orig/lib/tools/Makefile
+++ erlang-16.b-dfsg/lib/tools/Makefile
@@ -23,7 +23,7 @@
 # Macros
 # ----------------------------------------------------
 
-SUB_DIRECTORIES = c_src src doc/src examples priv emacs
+SUB_DIRECTORIES = c_src src examples priv emacs doc/src
 
 include vsn.mk
 VSN = $(TOOLS_VSN)
